### ISSUE TITLE
Weight decay is not used

### ISSUE THREAD

**Issue by nurlanov-zh** [eth-sri/paradox#1]
Hi,

Do you use L1 regularization in your training code `cert.py`? I can't find it.

Thanks for releasing the code, and for publishing a nice paper!

**Comment by rand0musername** [comment]
Hi, 

Sorry for the late reply and thank you for your interest in our work.

We implement regularization via `--reg` and `--reg_loss` parameters, and use L1 in our Table 2 runs (see `scripts/table2.sh`). You can find these flags defined in `args_factory.py` and used in `provable_training/utils.py`. 

Feel free to reopen the issue if you have further questions.
